src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -130,6 +130,10 @@ public static function getStyleDefinition($name)
      */
     public function setStyle($name)
     {
+        if ('symfony-style-guide' === $name) {
+            @trigger_error(sprintf('Style symfony-style-guide is deprecated since Symfony 4.1.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
The "box" style doesn't work on Windows `cmd.com` terminal, so we cannot deprecate `symfony-style-guide`.